logic/transaction_pool.js

```diff
@@ -513,7 +513,12 @@ TransactionPool.prototype.reindexQueues = function() {
 TransactionPool.prototype.processBundled = function(cb) {
 	const bundled = self.getBundledTransactionList(true, self.bundleLimit);
 
-	library.balancesSequence.add(
+	if (modules.loader.syncing()) {
```

Comment:
Why here and not in transport module? We should not allow receiving instead of processing, because the node will accumulate a lot of transactions during sync.

Reply:
We were discussing about that finally rejecting it because when the node enters into sync process due to a fork, it has to be able to receive and broadcast transactions. At least, this is how it works now. Otherwise, it will produce a behavioural change.